specs/cm/change-mgt-vocab.html

```diff
@@ -1,154 +1,127 @@
 <!DOCTYPE html>
 <html>
 <head>
-<meta charset='utf-8'/>
-<title>OSLC Change Management Version 3.0. Part 2: Vocabulary</title>
-<script src='https://raw.githack.com/oasis-tcs/tab-respec/master/builds/respec-oasis-common.js' async class='remove'></script>
-<script class='remove'>
-  var respecConfig = {
-    specStatus: "PSD",
-    revision: "01",
-    publishDate:  "2018-06-08",
-
-    license : "cc-by-4",
-
-    additionalLicenses : [{
-       licenseName : "Apache License 2.0",
-       licenseURI : "https://www.apache.org/licenses/LICENSE-2.0",
-     }],
-
-    edDraftURI: "https://raw.githack.com/oslc-op/oslc-specs/master/specs/cm/change-mgt-vocab.html",
-    thisVersion: "https://docs.oasis-open.org/oslc-op/cm/v3.0/ps01/change-mgt-vocab.html",
-    prevVersion: "http://docs.oasis-open.org/oslc-domains/cm/v3.0/cs02/part2-change-mgt-vocab/cm-v3.0-cs02-part2-change-mgt-vocab.html",
-    latestVersion: "https://docs.oasis-open.org/oslc-op/cm/change-mgt-vocab.html",
-
-    // the specification's short name, as in http://www.w3.org/TR/short-name/
-    shortName: "oslc-cm",
-    conformanceLabelPrefix: "CM",
-
-
-    // the document's citation label
-    citationLabel: "OSLC-CM-3.0-Part2",
-
-    // if your specification has a subtitle that goes below the main
-    // formal title, define it here
-    // subtitle   :  "an excellent document",
-
-    // if the specification's copyright date is a range of years, specify
-    // the start date here:
-    // copyrightStart: "2005"
-
-    // if there is a previously published draft, uncomment this and set its YYYY-MM-DD date
-    // and its maturity status
-    // previousPublishDate:  "1977-03-15",
-    // previousMaturity:  "WD",
-
-    // if there a publicly available Editor's Draft, this is the link
-    // edDraftURI:           "http://berjon.com/",
-
-    // if this is a LCWD, uncomment and set the end of its review period
-    // lcEnd: "2009-08-05",
-
-    // chairs and editors, add as many as you like
-    // only "name" is required
-    chairs : [ {
-      name : "Jim Amsden",
-      mailto : "jamsden@us.ibm.com",
-      company: "IBM",
-      companyURL: "http://www.ibm.com/"
-    }, {
-      name : "Andrii Berezovskyi",
-      mailto: "andriib@kth.se",
-      company: "KTH",
-      companyURL: "https://www.kth.se/en"
-    }],
-
-    editors:  [
+  <meta charset='utf-8'/>
+  <title>OSLC Change Management Version 3.0. Part 2: Vocabulary</title>
+
+  <script src='https://cdn.jsdelivr.net/gh/oasis-tcs/tab-respec@v2.1.6/builds/respec-oasis-common.min.js' async class='remove'></script>
+  <!-- <script src='http://127.0.0.1:9000/respec-oasis-common-2.1.6.js' async class='remove'></script> -->
+
+  <script class='remove'>
+    var filePath = "/change-mgt-vocab.html";
```

Comment:
relative URL shouldn't start with /

Reply:
this is a JavaScript variable, used only internally. I also expect @ndjc to be not quite thrilled with my workaround :)